.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "ruby-definitions",
+	"image": "ghcr.io/rails/devcontainer/images/ruby:3.3.0",
```

Comment:
Any reason we're not using 3.4.7 like https://github.com/Shopify/autotuner/blob/main/.devcontainer/devcontainer.json#L6?